### PR TITLE
Gherkin scenarios for list pages 

### PR DESCRIPTION
**Pull request by matejnesuta**
Related to #6474.

I have also updated existing tests with some additional checks.

### CODE REVIEW

File: frontend/cypress/integration/featureFiles/services.feature [hunk 1]
```diff
@@ -122,4 +122,25 @@ Feature: Kiali Services page
 
   @multi-cluster
   Scenario: The column related to cluster name should be visible
+    When user selects the "bookinfo" namespace
     Then the "Cluster" column "appears"
+    And an entry for "east" cluster should be in the table
+    And an entry for "west" cluster should be in the table
+
+  # inspired by this: https://github.com/kiali/kiali/pull/5998#pullrequestreview-1383754665
+  @services-page
```
Comment (ScriptingShrimp) [inline frontend/cypress/integration/featureFiles/services.feature:131]: we can omit this tag in favor of `@services` at the  first line

File: frontend/cypress/integration/featureFiles/services.feature [hunk 2]
```diff
@@ -122,4 +122,25 @@ Feature: Kiali Services page
...
+  @skip
+  @multi-cluster
+  Scenario: Services from both clusters should have a validation
+    When user selects the "bookinfo" namespace
+    Then all of the default "bookinfo" services from both clusters should be "healthy" 
+
+  @services-page
```
Comment (ScriptingShrimp) [inline frontend/cypress/integration/featureFiles/services.feature:138]: we can omit this tag in favor of `@services` at the  first line

File: frontend/cypress/integration/featureFiles/workloads.feature [hunk 3]
```diff
@@ -123,6 +123,19 @@ Feature: Kiali Workloads page
     And the health status of the workload should be "Degraded"
 
   @multi-cluster
-  @skip
+  @workloads-page
```
Comment (ScriptingShrimp) [inline frontend/cypress/integration/featureFiles/workloads.feature:126]: we can omit this tag in favor of `@workloads` at the  first line

File: frontend/cypress/integration/featureFiles/workloads.feature [hunk 4]
```diff
@@ -123,6 +123,19 @@ Feature: Kiali Workloads page
...
   Scenario: The column related to cluster name should be visible
-    Then the "Cluster" column "appears"
+    When user selects the "bookinfo" namespace
+    Then the "Cluster" column "appears" 
+    And an entry for "east" cluster should be in the table
+    And an entry for "west" cluster should be in the table
+
+  @workloads-page
```
Comment (ScriptingShrimp) [inline frontend/cypress/integration/featureFiles/workloads.feature:133]: we can omit this tag in favor of `@workloads` at the first line

**Review by ScriptingShrimp — CHANGES_REQUESTED** [review]
I think I could resolve conflicts better. Nitpicked few tags, otherwise LGTM

**Review by hhovsepy — APPROVED** [review]
lgtm

**Pull request merged**